src/LibUsbDotNet/Generated/ControlSetup.cs

```diff
@@ -34,37 +34,34 @@
 
 namespace LibUsbDotNet
 {
-    /// <summary>
-    ///  Setup packet for control transfers.
-    /// </summary>
```

Comment:
In the newer `libusb.h` there are preprocessor directives between the doc comment and the struct definition, which I believe is what's causing this to get dropped.